### DISCUSSION TITLE
Installing/Upgrading to the latest commit

### DISCUSSION THREAD

**Discussion by anon241242** [Serene-Arc/bulk-downloader-for-reddit#687] [Q&A] [answered]
Hello all,
can someone tell me how to install the latest commit? 
Its not "released", so I cant upgrade the normal way: ```python3 -m pip install bdfr --upgrade```
Current latest commit: https://github.com/aliparlakci/bulk-downloader-for-reddit/pull/683
I am aware that this is a temporary fix, but I want to utilise it.

Excuse me If there is an obvious solution. I am very new to Python and Github

**Comment by Serene-Arc** [comment, marked as answer]
The solution is to install directly from the git repository.

`python3 -m pip install -U git+https://github.com/aliparlakci/bulk-downloader-for-reddit.git`
